models/stg_quickbooks.yml

```diff
@@ -223,7 +223,7 @@ models:
         description: Reference number for the transaction.
       - name: vendor_id
         description: Reference to the vendor the bill is linked with.
-      - name: due_date_at
+      - name: due_date
```

Comment:
Ah yes I switched it back to make it standard with the other models, but then realized it's called upon downstream so I meant to revert it to `due_date_at`. Although, if we are making a PR in the transforms now, perhaps we should make it `due_date` and make the appropriate changes downstream.

I actually was wondering why it was changed to due_date_at, since it just gets renamed to due_date in transforms where it gets called.
[`        bill_link.due_date_at as due_date,`](https://github.com/fivetran/dbt_quickbooks/blob/9bd1a21f22bbb73d573cb9587c0c30416d44e23f/models/intermediate/int_quickbooks__bill_join.sql#L75)

@fivetran-joemarkiewicz Any thoughts? I don't see the reason why it was renamed to due_date_at, in `stg_quickbooks__bill` only

Reply:
This likely was just human error for why this one was named differently. Truthfully, I would be cautious if you are changing this as it will likely cause more scope creep than you think as you called out it is leveraged downstream. Additionally, if you change the name of this field it will be reflected as a breaking change in the source.

We can chat more about this during standup.

Reply:
Added to the existing PR we now have for small fixes.
https://github.com/fivetran/dbt_quickbooks/issues/101